test/fixtures/workflow_executions.yml

```diff
@@ -602,6 +604,7 @@ automated_example_completed:
   run_id: "my_run_id_6"
   submitter_id: <%= ActiveRecord::FixtureSet.identify(:project1_automation_bot, :uuid) %>
   namespace_id: <%= ActiveRecord::FixtureSet.identify(:project1_namespace, :uuid) %>
+  name: "autmoated_example_completed"
```

Comment:
**nitpick:** Typo: `autmoated_example_completed` to `automated_example_completed`

Reply:
Thanks, fixed in 4ffcfa40b046a408da6417069e63feb2f0df042e